api/v1/controllers/collectors.js

```diff
@@ -26,9 +26,58 @@ const doPatch = require('../helpers/verbs/doPatch');
 const doPut = require('../helpers/verbs/doPut');
 const u = require('../helpers/verbs/utils');
 const httpStatus = require('../constants').httpStatus;
+const decryptSGContextValues = require('../../../utils/cryptUtils')
+                                .decryptSGContextValues;
+const encrypt = require('../../../utils/cryptUtils').encrypt;
+const GlobalConfig = require('../../../db/index').GlobalConfig;
```

Comment:
You should get the globalConfig model from api/v1/helpers/nouns/globalconfig.js. There is no need to create a dependency between this one and db index.